### PR TITLE
Bump sbt-bin to 1.8.0

### PR DESCRIPTION
**Pull request by kerkmann**
Just a version bump, see: https://github.com/sbt/sbt/releases/tag/v1.8.0 :)

### CODE REVIEW

**Comment by Flowdalic** [comment]
Thanks for your contribution.

Please keep the existing version of the package and do not add new packages with stable keywords. Also, make sure that your commit message contains an signed-off git trailer.

**Comment by kerkmann** [comment]
Thanks, for your comment @Flowdalic ! :)

Yeah, I've seen the signe-off too late, will do that in the future. ^^"

What do you mean exactly `do not add new packages with stable keywords`?
As far as I know, the version `1.8.0` is a stable version.

And how can I add an unstable version? :)

Please reach out to me, so I can change it and know it for the future. :3

**Comment by Flowdalic** [comment]
Instead of
`KEYWORDS="amd64"`
new packages are added with unstable keywords
`KEYWORDS="~amd64"`. See also devmanual.gentoo.org

**Comment by kerkmann** [comment]
Okay, I see. I think I got it. :)

File: dev-java/sbt-bin/sbt-bin-1.8.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (Flowdalic) [inline dev-java/sbt-bin/sbt-bin-1.8.0.ebuild:4]: I think we could also easily bump the EAPI to 8.

Comment (kerkmann) [reply dev-java/sbt-bin/sbt-bin-1.8.0.ebuild]: Change it in the commit [6ea9898](https://github.com/gentoo/gentoo/pull/28474/commits/6ea98988f346aa38e62a5c4078283aebb2338d75), thanks. :3

File: dev-java/sbt-bin/sbt-bin-1.8.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+DESCRIPTION="sbt, a build tool for Scala"
+HOMEPAGE="https://scala-sbt.org"
+SRC_URI="https://github.com/sbt/sbt/releases/download/v${PV}/${PN/-bin}-${PV}.tgz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=virtual/jre-1.7
```
[the remark this answers is not in the document]
Comment (kerkmann) [reply dev-java/sbt-bin/sbt-bin-1.8.0.ebuild]: Change it in the commit [6ea9898](https://github.com/gentoo/gentoo/pull/28474/commits/6ea98988f346aa38e62a5c4078283aebb2338d75), thanks. :3

**Comment by Flowdalic** [comment]
FYI, the canonical format of the commit subject should be "`<cat>/<pgk>: <description>`" I took care of that while merging this. Thanks.

**Pull request closed without merging**